build.js

```diff
@@ -62,4 +61,3 @@ writeFileSync(
   JSON.stringify(cjspkg, null, '  '),
   'utf8'
 )
-execSync('cp -r node_modules/string-comparison/lib/main dist/cjs')
```

Comment:
Remove due to https://github.com/ndaidong/article-parser/issues/206#issuecomment-1037949543